airtime_mvc/application/common/FileDataHelper.php

```diff
@@ -68,4 +68,369 @@ public static function getFileExtensionFromMime($mime)
+          'ý' => 'y',
+          'þ' => 'b',
+          'ÿ' => 'y',
+          ' ' => '_',
+          '(' => '_',
+          ')' => '_',
+          '[' => '_',
+          ']' => '_'
+        );
+        return strtr($string, $normal);
+    }
+
+    /**
+     * Render image
+     * Used in API
+     *
+     * @param string $file
+     * @param string $size
+     */
+    public static function renderImage($file)
+    {
+          $im = @imagecreatefromjpeg($file);
+          header('Content-Type: image/jpeg');
+          $img = $im;
+          imagejpeg($img);
+          imagedestroy($img);
+    }
+
+    /**
+     * Render Data URI
+     * Used in API
+     *
+     * @param string $dataFile
+     */
+    public static function renderDataURI($dataFile)
+    {
+          if($filecontent = file_get_contents($dataFile) !== false){
+               $image = @file_get_contents($dataFile);
+               $image = base64_encode($image);
+               if (!$image || $image === '') {
+                   return;
+               }
+               $blob = base64_decode($image);
+               $f = finfo_open();
+               $mime_type = finfo_buffer($f, $blob, FILEINFO_MIME_TYPE);
+               finfo_close($f);
+               header("Content-Type: " . $mime_type);
+               echo $blob;
+          } else {
+              return;
+          }
+    }
+
+    /**
+     * Resize Image
+     *
+     * @param string $orig_filename
+     * @param string $converted_filename
+     * @param string $ext
+     * @param string $size      Default: 500
+     * @param string $quality   Default: 75
+     *
+     */
+    public static function resizeImage($orig_filename, $converted_filename, $ext, $size=500, $quality=75)
+    {
+          $get_cont = file_get_contents($orig_filename);
+          if ($ext == "png") {
+              $im = @imagecreatefrompng($get_cont);
+          } elseif ($ext == "gif") {
+              $im = @imagecreatefromgif($get_cont);
+          } else {
+              $im = @imagecreatefromjpeg($get_cont);
+          }
+
+          if ($size){
+            $im = imagescale($im , $size);
+          }
+
+          if(!$im) {
+              $im  = imagecreatetruecolor(150, 30);
+              $bgc = imagecolorallocate($im, 255, 255, 255);
+              $tc  = imagecolorallocate($im, 0, 0, 0);
+              imagefilledrectangle($im, 0, 0, 150, 30, $bgc);
+              imagestring($im, 1, 5, 5, 'Error loading ' . $imgname, $tc);
+          }
+
+          $img = $im;
+          imagejpeg($img, $converted_filename, $quality);
+          imagedestroy($img);
+    }
+
+    /**
+     * Convert image to Data URI
+     *
+     * @param string $orig_filename
+     * @param string $conv_filename
+     */
+    public static function imgToDataURI($orig_filename, $conv_filename)
+    {
+          $file = file_get_contents($orig_filename);
+          $Image = 'data:image/jpeg;charset=utf-8;base64,'.base64_encode($file);
+          $base64 = @$Image;
```

Comment:
This `@` only seems to ignore errors when assigning one string to another variable. Please remove it.